COVID_HUNGARY_REPORT.wls

```diff
@@ -234,18 +117,7 @@ sol=modelSolver[beta, initialValuesImod, alpha, alpha2, gammaa, gammas, gammah,
 initialValuesImod=Round[modelGenerator[beta, init, alpha, alpha2, gammaa, gammas, gammah, gammac, gammacr, ageDistribution, mu, ksi, asymp, pA, h][[1]]/.sol/.t->24][[1,1]]
 
 
-(* ::Subsubsection:: *)
-(*Functions*)
-
-
-mortalityCumulative=Table[Sum[d[j][t],{j,1,i}],{i,1,nAge}];
-cumulative=Table[Sum[c[j][t],{j,1,i}],{i,1,nAge}];
-
-
-sumOfI[t_]:=Evaluate[Sum[ia1[i][t]+is1[i][t]+ia2[i][t]+is2[i][t]+ia3[i][t]+is3[i][t]+ip[i][t]+icr[i][t],{i,1,nAge}]/.sol];
-
-
-(* ::Section:: *)
+(* ::Section::Closed:: *)
 (*Scenarios post-lockdown*)
```

Comment:
My idea about "sectionization":
Scenarios post-lockdown
├── Plots for Figure 6.
 |       ├── Plots for hospitalization
 |        |       ├── R=1.65  
 |        |       ├── R=1.32  
 |        |       └── R=1.1
 |       └── Plots for incidence
 |                ├── R=1.65  
 |                ├── R=1.32  
 |                └── R=1.1
└── Plots for Figure 7.
         ├── Mortality 
          |       ├── R=1.65  
          |       ├── R=1.32  
          |       └── R=1.1
         └── Recovered 
                  ├── R=1.65  
                  ├── R=1.32  
                  └── R=1.1
This means (since last Heading level in Mma is level 6), that 
top level header (Scenarios post-lockdown) is level 3, which gives immediately, that all top level headers (Initialization, age-specific analysis, ...) should be also level 3!